modules/javafx.web/src/main/java/javafx/scene/web/WebView.java

```diff
@@ -692,6 +697,60 @@ public String getName() {
         return this.fontSmoothingType;
     }
 
+    /**
+     * Specifies the background color of the web page.
+     *
+     * <p>With this property, the {@code WebView} control's background
+     * can have any solid color, including some or complete
+     * level of transparency.
+     *
+     * <p>However, if the HTML content being loaded sets its own
```

Comment:
> I looked at the generated javadoc and this will read better without a paragraph break.

When you remove the paragraph break, please also remove the blank line to make it clearer to future readers of the code that it is intended to be part of the same paragraph as the previous sentence.

Reply:
Done